configs/TotalSegmentator/radiomicsFeaturesMaps.csv

```diff
@@ -10,19 +10,20 @@
 8,SurfaceArea,shape,Calculated using Marching Cubes,IBSI,C0JK,Surface area,UCUM,mm2,square millimeter
 9,SurfaceVolumeRatio,shape,,IBSI,2PR5,Surface to volume ratio,UCUM,/mm,per millimeter
 10,VoxelVolume,shape,,IBSI,YEKZ,Approximate volume,UCUM,mm3,cubic millimeter
-11,10Percentile,firstorder,,IBSI,QG58,10th percentile,UCUM,1,no units
-12,90Percentile,firstorder,,IBSI,8DWT,90th percentile,UCUM,1,no units
-13,Energy,firstorder,,IBSI,N8CA,Energy,UCUM,1,no units
+11,10Percentile,firstorder,,IBSI,QG58,10th percentile,UCUM,[hnsf'U],Hounsfield Unit
+12,90Percentile,firstorder,,IBSI,8DWT,90th percentile,UCUM,[hnsf'U],Hounsfield Unit
+13,Energy,firstorder,,IBSI,N8CA,Energy,UCUM,[hnsf'U]2,square Hounsfield Unit
 14,Entropy,firstorder,,IBSI,TLU2,Intensity histogram entropy,UCUM,1,no units
-15,InterquartileRange,firstorder,,IBSI,SALO,Interquartile range,UCUM,1,no units
-16,Maximum,firstorder,,IBSI,84IY,Maximum grey level,UCUM,1,no units
-17,MeanAbsoluteDeviation,firstorder,,IBSI,4FUA,Mean absolute deviation,UCUM,1,no units
-18,Mean,firstorder,,IBSI,Q4LE,Mean,UCUM,1,no units
-19,Median,firstorder,,IBSI,Y12H,Median,UCUM,1,no units
-20,Minimum,firstorder,,IBSI,1GSF,Minimum grey level,UCUM,1,no units
-21,Range,firstorder,,IBSI,2OJQ,Range,UCUM,1,no units
-22,RobustMeanAbsoluteDeviation,firstorder,,IBSI,1128,Robust mean absolute deviation,UCUM,1,no units
-23,RootMeanSquared,firstorder,,IBSI,5ZWQ,Root mean square,UCUM,1,no units
+15,InterquartileRange,firstorder,,IBSI,SALO,Interquartile range,UCUM,[hnsf'U],Hounsfield Unit
+16,Maximum,firstorder,,IBSI,84IY,Maximum grey level,UCUM,[hnsf'U],Hounsfield Unit
+17,MeanAbsoluteDeviation,firstorder,,IBSI,4FUA,Mean absolute deviation,UCUM,[hnsf'U],Hounsfield Unit
+18,Mean,firstorder,,IBSI,Q4LE,Mean,UCUM,[hnsf'U],Hounsfield Unit
+19,Median,firstorder,,IBSI,Y12H,Median,UCUM,[hnsf'U],Hounsfield Unit
+20,Minimum,firstorder,,IBSI,1GSF,Minimum grey level,UCUM,[hnsf'U],Hounsfield Unit
+21,Range,firstorder,,IBSI,2OJQ,Range,UCUM,[hnsf'U],Hounsfield Unit
+22,RobustMeanAbsoluteDeviation,firstorder,,IBSI,1128,Robust mean absolute deviation,UCUM,[hnsf'U],Hounsfield Unit
+23,RootMeanSquared,firstorder,,IBSI,5ZWQ,Root mean square,UCUM,[hnsf'U],Hounsfield Unit
 24,Skewness,firstorder,,IBSI,KE2A,Skewness,UCUM,1,no units
 25,Uniformity,firstorder,,IBSI,BJ5W,Intensity histogram uniformity,UCUM,1,no units
-26,Variance,firstorder,,IBSI,ECT3,Variance,UCUM,1,no units
+26,Variance,firstorder,,IBSI,ECT3,Variance,UCUM,[hnsf'U]2,square Hounsfield Unit
+27,Kurtosis,firstorder,,IBSI,IPH6,Kurtosis,UCUM,[hnsf'U]2,square Hounsfield Unit
```

Comment:
From what I found, Kurtosis has no units, so it would be [UCUM,1,no units]

Reply:
The formula for kurtosis is 
![image](https://github.com/ImagingDataCommons/Cloud-Resources-Workflows/assets/115020590/b1c9340c-f7e5-493c-bda2-4df20910c9a1)
By this, it is equivalent to X^2.

Reply:
I spoke too soon. you are correct. It will be a dimensionless.